Utilities/bootstrap

```diff
@@ -369,6 +373,32 @@ def install(args):
         tscclibc_include_dir_dest = os.path.join(dest, "TSCclibc")
         dir_util.copy_tree(tscclibc_include_dir, tscclibc_include_dir_dest)
 
+    # Install libSwiftPMDataModel if an install directory was provided.
```

Comment:
This currently duplicates the libSwiftPM code path.  There were enough differences that a combined version had a lot of conditionals, but it might be worth trying to combine them.